tools/src/main/java/org/bitcoinj/tools/BuildCheckpoints.java

```diff
@@ -46,12 +51,31 @@
  * to a file which is then signed with your key.
  */
 public class BuildCheckpoints {
-
-    private static final NetworkParameters PARAMS = MainNetParams.get();
-    private static final File PLAIN_CHECKPOINTS_FILE = new File("checkpoints");
-    private static final File TEXTUAL_CHECKPOINTS_FILE = new File("checkpoints.txt");
+    private static NetworkParameters params;
 
     public static void main(String[] args) throws Exception {
+        OptionParser parser = new OptionParser();
+        OptionSpec<NetworkEnum> netFlag = parser.accepts("net").withRequiredArg().ofType(NetworkEnum.class).defaultsTo(NetworkEnum.MAIN);
```

Comment:
There is a default - see last clause of statement.  That means that the user can either omit the --net option entirely, but if they do supply it they have to provide an argument.